### PR TITLE
koord-scheduler: exclude DefaultQuota and SystemQuota when updating quota tree

### PR DESCRIPTION
**Pull request by xulinfei1996**
Signed-off-by: xulinfei.xlf <xulinfei.xlf@alibaba-inc.com>

### Ⅰ. Describe what this PR does
SystemQuotaInfo and DefaultQuotaInfo should not appear in runtimeQuotaCalculator.

### V. Checklist

- [ ] I have written necessary docs and comments
- [ ] I have added necessary unit tests and integration tests
- [ ] All checks passed in `make test`

### CODE REVIEW

File: pkg/scheduler/plugins/elasticquota/core/group_quota_manager.go [hunk 1]
```diff
@@ -145,6 +145,10 @@ func (gqm *GroupQuotaManager) updateGroupDeltaRequestTopoRecursiveNoLock(deltaRe
 		newSubLimitReq := curQuotaInfo.getLimitRequestNoLock()
 		deltaReq = quotav1.Subtract(newSubLimitReq, oldSubLimitReq)
 
+		if curQuotaInfo.Name == extension.SystemQuotaName || curQuotaInfo.Name == extension.DefaultQuotaName {
```
Comment (eahydra) [inline pkg/scheduler/plugins/elasticquota/core/group_quota_manager.go:148]: as we talked,  if the curQuotaInfo is one of (extension.SystemQuotaName, extension.DefaultQuotaName),  we should update the requested of Quota. And refactor the function, make the function clear.

Comment (xulinfei1996) [reply pkg/scheduler/plugins/elasticquota/core/group_quota_manager.go]: fixed

**Review by eahydra — APPROVED** [review]
/lgtm

**Review by buptcozy — COMMENTED** [review]
/lgtm

**Pull request merged**